qc_generator/qc_generator.cwl

```diff
@@ -0,0 +1,957 @@
+class: Workflow
```

Comment:
is this a new workflow or is this the new access_bam_qc.cwl ?

Reply:
it's just `access_bam_qc.cwl`. I just renamed it to make if hopefully clearer

Reply:
if you think there is a better name, let me know

Reply:
👍